docs/source/config_file.rst

```diff
@@ -65,234 +65,346 @@ unfortunate, and is subject to change in future versions.
+    an unfollowed import is automatically given a type of ``Any``).
 
-  **Note:** This was False by default
-  in mypy versions earlier than 0.600.
+``disallow_any_expr`` (boolean, default False)
+    Disallows all expressions in the module that have type ``Any``.
 
-- ``disallow_any_unimported`` (Boolean, default false) disallows usage of types that come
-  from unfollowed imports (such types become aliases for ``Any``).
+``disallow_any_decorated`` (boolean, default False)
+    Disallows functions that have ``Any`` in their signature after decorator transformation.
 
-- ``disallow_any_expr`` (Boolean, default false) disallows all expressions in the module
-  that have type ``Any``.
+``disallow_any_explicit`` (boolean, default False)
+    Disallows explicit ``Any`` in type positions such as type annotations and generic
+    type parameters.
 
-- ``disallow_any_decorated`` (Boolean, default false) disallows functions that have ``Any``
-  in their signature after decorator transformation.
+``disallow_any_generics`` (boolean, default False)
+    Disallows usage of generic types that do not specify explicit type parameters.
 
-- ``disallow_any_explicit`` (Boolean, default false) disallows explicit ``Any`` in type
-  positions such as type annotations and generic type parameters.
+``disallow_subclassing_any`` (boolean, default False)
+    Disallows subclassing a value of type ``Any``.
 
-- ``disallow_any_generics`` (Boolean, default false) disallows usage of generic types that
-  do not specify explicit type parameters.
 
-- ``disallow_incomplete_defs`` (Boolean, default false) disallow defining
-  functions with incomplete type annotations.  See
-  :ref:`--disallow-incomplete-defs <disallow-incomplete-defs>` option.
+Untyped definitions and calls
+-----------------------------
 
-- ``disallow_subclassing_any`` (Boolean, default False) disallows
-  subclassing a value of type ``Any``.  See
-  :ref:`--disallow-subclassing-any <disallow-subclassing-any>` option.
+For more information, see the :ref:`untyped definitions and calls <untyped-definitions-and-calls>`
+section of the command line docs.
 
-- ``disallow_untyped_calls`` (Boolean, default False) disallows
-  calling functions without type annotations from functions with type
-  annotations.
+``disallow_untyped_calls`` (boolean, default False)
+    Disallows calling functions without type annotations from functions with type
+    annotations.
 
-- ``disallow_untyped_defs`` (Boolean, default False) disallows
-  defining functions without type annotations or with incomplete type
-  annotations.
+``disallow_untyped_defs`` (boolean, default False)
+    Disallows defining functions without type annotations or with incomplete type
+    annotations.
 
-- ``check_untyped_defs`` (Boolean, default False) type-checks the
-  interior of functions without type annotations.
+``disallow_incomplete_defs`` (boolean, default False)
+    Disallow defining functions with incomplete type annotations. 
```

Comment:
Can you be consistent about "disallow" vs. "disallows"? I think most places use "disallows", "type-checks", reports" etc., so I'd go with that.

Reply:
Whoops, thanks for the catch! Fixed.